TESTING-CHECKLIST.md

```diff
@@ -0,0 +1,81 @@
+## Testing
+
+### Login
+
+- [ ] Logout
+- [ ] Login with wrong password fails
+- [ ] Login with correct password
+
+### Sync
+
+- [ ] Create new note appears in other device
+- [ ] Changes to new note sync to other device
+- [ ] Changes to new note sync _from_ other device
+- [ ] New tag immediately syncs to other device
+- [ ] New tag immediately syncs _from_ other device
+- [ ] Removed tag immediately syncs to other device
+- [ ] Removed tag immediately syncs _from_ other device
+- [ ] Note publishes with link
+- [ ] Note unpublishes
+- [ ] Note publish change syncs _from_ other device (visible with dialog open)
+- [ ] Markdown setting syncs _from_ other device
+- [ ] Preview mode disappears/reappears when receiving remote changes to markdown setting
+- [ ] Note pinning syncs immediately from either direction
+- [ ] Note pinning works regardless if clicking in list view or from note info (and syncs after the editor is opened or the app is resumed from being in the background)
+- [ ] Viewing history on one device leaves note unchanged on other device
+- [ ] Restoring history immediately syncs note from both directions
+- [ ] After disabling network connectivity and making changes, selecting Log Out triggers an `Unsynced Notes Detected` alert
+- [ ] After going back online, changes sync
+
+### Note editor
+
+- [ ] Can preview markdown with Preview tab (👁 button on landscape tablet)
+- [ ] Can flip to edit mode with Edit tab (👁 button on landscape tablet)
+- [ ] Can toggle sidebar (in landscape tablet)
+- [ ] Tapping the `Insert checklist` icon inserts a checklist
+- [ ] Typing `- [x]` creates a checked checklist item
+- [ ] Typing `- [ ]` created an unchecked checklist item
+- [ ] Typing `-` creates a list
+- [ ] All list bullet types render to markdown lists
+- [ ] Added URL is linkified
+- [ ] Tapping on link opens it in browser
+
+### Tags & search
+
+- [ ] Can filter by tag when clicking on tag in tag drawer
+- [ ] Can add tag to note and have it appear in filtered tag view when previously not in filter
+- [ ] Searching in the search field does a global search
+- [ ] Searching in the search field highlights matches in note list
+- [ ] Searching in the search field highlights matches in the note editor
+- [ ] Clearing the search field immediately updates filtered notes
+- [ ] Clicking on different tags or `All Notes` or `Trash` immediately updates filtered notes
+- [ ] Can search by keyword, filtered instantly
```

Comment:
The "filtered instantly" part might be a little misleading.  Android requires you to submit thee search before the filtered note list is shown as search results.  Electron and iOS show results as you type.  Is that what you mean by "filtered instantly" here?

Reply:
Ah, yep, this snuck in from the wording used in the Electron checklist. I'll update that, thanks!